package/utils/util-linux/Makefile

```diff
@@ -35,6 +35,31 @@ define Package/util-linux/Default
   URL:=http://www.kernel.org/pub/linux/utils/util-linux/
 endef
 
+define Package/util-linux-busybox
+$(call Package/util-linux/Default)
+  TITLE:=Full util-linux commands instead of busybox replacements
```

Comment:
A typo? Or is `util-linux-busybox` indeed replacing busybox applets? That'd be very confusing...